context/app/templates/pages/base.html

```diff
@@ -28,9 +32,9 @@
         </div>
       </div>
     </div>
-
-    {% include 'fragments/footer.html' %}
-
-    {% block extra_scripts %}{% endblock %}
+    <footer id="react-footer"></footer>
+    {% block extra_scripts %}
+      <script src="{{ url_for('static', filename='public/bundle.js') }}"></script>
```

Comment:
Should this be outside the extra_scripts block? Won't we want bundle.js on every page?